# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../libdnet/gcc4.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
diff -Nur libdnet-1.10.orig/python/dnet.c libdnet-1.10/python/dnet.c
--- libdnet-1.10.orig/python/dnet.c	2005-02-26 01:06:02.000000000 +0200
+++ libdnet-1.10/python/dnet.c	2005-11-01 14:12:38.000000000 +0200
@@ -2552,7 +2552,7 @@
   PyObject *__pyx_3 = 0;
   unsigned short __pyx_4;
   Py_INCREF(__pyx_v_self);
-  ((PyObject*)__pyx_v_next) = Py_None; Py_INCREF(((PyObject*)__pyx_v_next));
+  __pyx_v_next = Py_None; Py_INCREF(((PyObject*)__pyx_v_next));
 
   /* "/misc/projects/libdnet/python/./dnet.pyx":628 */
   __pyx_1 = (((struct __pyx_obj_4dnet___addr_ip4_iter *)__pyx_v_self)->cur <= ((struct __pyx_obj_4dnet___addr_ip4_iter *)__pyx_v_self)->max);
@@ -2564,7 +2564,7 @@
     Py_DECREF(__pyx_2); __pyx_2 = 0;
     if (!__Pyx_TypeTest(__pyx_3, __pyx_ptype_4dnet_addr)) {__pyx_filename = __pyx_f[0]; __pyx_lineno = 629; goto __pyx_L1;}
     Py_DECREF(((PyObject *)__pyx_v_next));
-    ((PyObject *)__pyx_v_next) = __pyx_3;
+    __pyx_v_next = __pyx_3;
     __pyx_3 = 0;
 
     /* "/misc/projects/libdnet/python/./dnet.pyx":630 */
